### PR TITLE
added typescript-validation-schema plugin to website

### PR DESCRIPTION
**Pull request by Code-Hex**
- I created this PR with according
  - https://github.com/dotansimha/graphql-code-generator/pull/1326
  - https://github.com/dotansimha/graphql-code-generator/pull/7378
- Plugin: https://github.com/Code-Hex/graphql-codegen-typescript-validation-schema
- issues: https://github.com/dotansimha/graphql-code-generator/issues/7437

### CODE REVIEW

**Comment by charlypoly** [comment]
Preview here: https://graphql-code-generator-fgblanvd4-theguild.vercel.app/plugins/typescript-validation-schema

**Comment by Code-Hex** [comment]
@charlypoly Great 👏 
https://graphql-code-generator-fgblanvd4-theguild.vercel.app/plugins/typescript-validation-schema

File: website/docs-templates/graphql-codegen-typescript-validation-schema.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (charlypoly) [inline website/docs-templates/graphql-codegen-typescript-validation-schema.md:1]: @Code-Hex This file should be renamed to `typescript-validation-schema,md`

**Review by charlypoly — CHANGES_REQUESTED** [review]

**Comment by charlypoly** [comment]
@Code-Hex better! https://graphql-code-generator-git-fork-code-hex-add-ty-b477d8-theguild.vercel.app/plugins/typescript-validation-schema

**Review by charlypoly — APPROVED** [review]

**Pull request merged**